mozillians/settings.py

```diff
@@ -115,6 +115,7 @@
 SECURE_BROWSER_XSS_FILTER = config("SECURE_BROWSER_XSS_FILTER", default=True, cast=bool)
 ENABLE_REFERRER_HEADER = config("ENABLE_REFERRER_HEADER", default=True, cast=bool)
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+SEECURE_SSL_REDIRECT == config("SECURE_SSL_REDIRECT", default=True, cast=bool)
```

Comment:
`SEECURE_SSL_REDIRECT ==`

I think you might want one less `E` and one less `=` :stuck_out_tongue:

Reply:
hmmm